velox/expression/tests/utils/ArgumentTypeFuzzer.cpp

```diff
@@ -35,6 +38,34 @@ std::optional<TypeKind> baseNameToTypeKind(const std::string& typeName) {
   return tryMapNameToTypeKind(kindName);
 }
 
+void ArgumentTypeFuzzer::determineUnboundedIntegerVariables() {
+  for (const auto& arg : signature_.argumentTypes()) {
+    if (arg.baseName() == "decimal" || arg.baseName() == "DECIMAL") {
+      auto precision = arg.parameters()[0].baseName();
+      auto scale = arg.parameters()[1].baseName();
```

Comment:
Sometimes precision and scale does not come in pair. I wonder if this works for case like decimal(precision, 6) or decimal(12, scale).